### ISSUE TITLE
Error when giving item

### ISSUE THREAD

**Issue by MrBretze** [Minestom/Minestom#687]
I have an error when a use a the give command;
`give @p minecraft:oak_sign{BlockEntityTag:{Text1:'{"text":"","clickEvent":{"action":"run_command","value":"say YOLO"}}',Text2:'["",{"text":"Zebi la "},{"text":"mouche","bold":true,"color":"dark_red"}]'},display:{Name:'{"text":"Y\\u00e9"}'}}`
```
[Ms-worker-2] [00:26:55] (ExceptionHandler.handleException) - ERROR - Error: java.lang.IndexOutOfBoundsException: Range [7, 7 + -852188400) out of bounds for length 2097151: java.lang.IndexOutOfBoundsException: Range [7, 7 + -852188400) out of bounds for length 2097151
	at java.base/jdk.internal.util.Preconditions.outOfBounds(Preconditions.java:64)
	at java.base/jdk.internal.util.Preconditions.outOfBoundsCheckFromIndexSize(Preconditions.java:82)
	at java.base/jdk.internal.util.Preconditions.checkFromIndexSize(Preconditions.java:361)
	at java.base/java.util.Objects.checkFromIndexSize(Objects.java:411)
	at java.base/java.nio.DirectByteBuffer.slice(DirectByteBuffer.java:255)
	at java.base/java.nio.DirectByteBuffer.slice(DirectByteBuffer.java:40)
	at net.minestom.server.utils.binary.BinaryBuffer.asByteBuffer(BinaryBuffer.java:144)
	at net.minestom.server.utils.PacketUtils.readPackets(PacketUtils.java:190)
	at net.minestom.server.network.player.PlayerSocketConnection.processPackets(PlayerSocketConnection.java:102)
	at net.minestom.server.network.socket.Worker.lambda$run$0(Worker.java:69)
	at java.base/sun.nio.ch.SelectorImpl.processReadyEvents(SelectorImpl.java:294)
	at java.base/sun.nio.ch.EPollSelectorImpl.processEvents(EPollSelectorImpl.java:195)
	at java.base/sun.nio.ch.EPollSelectorImpl.doSelect(EPollSelectorImpl.java:135)
	at java.base/sun.nio.ch.SelectorImpl.lockAndDoSelect(SelectorImpl.java:129)
	at java.base/sun.nio.ch.SelectorImpl.select(SelectorImpl.java:161)
	at net.minestom.server.network.socket.Worker.run(Worker.java:58)
```

**Comment by TheMode** [comment]
Are you using latest?

**Comment by MrBretze** [comment]
Yes

**Comment by TheMode** [comment]
The lines do not correspond to anything so I highly doubt it.

**Comment by TheMode** [comment]
I assume fixed by f3f6da3127ef97c75ae91515f3d400441acd7178
